### ISSUE TITLE
uninstallable Bioconductor packages in this repository

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/bioconductor_release#1]
Here are the 11 Bioconductor packages that don't install on this image as of 1/24/2019, and the installation log showing errors. Most are various compilation errors, a couple need X11 which seems unnecessary to support, and one depends on an archived CRAN package. I did the testing with [this gist](https://gist.github.com/lwaldron/3b002e72b4e99fc093f8dace4ab38bf6).

```
"bgx" FALSE
"BiGGR" FALSE
"ccfindR" FALSE
"dSimer" FALSE
"EasyqpcR" FALSE
"flipflop" FALSE
"flowQB" FALSE
"mcaGUI" FALSE
"BiocSklearn" FALSE
"OMICsPCA" FALSE
"PAA" FALSE
"xps" FALSE
```
Note that the log file still contains errors for `PAA` and `BiocSklearn`, but these have been or are about to be resolved (`PAA` by contacting the maintainer of the archived CRAN package that it depends on, `BiocSklearn` by adding pip install commands for sklearn and pandas)

[install_log.txt](https://github.com/waldronlab/bioconductor_release/files/2791895/install_log.txt)

**Comment by vjcitn** [comment]
Thanks for doing this!  For BiocSklearn, can we add something like pip install sklearn?  In the log I found

Error in py_module_import(module, convert = convert) : 
  ImportError: No module named sklearn

**Comment by lwaldron** [comment]
Yes, done, will close once the Docker build is tested and I've confirmed it
installs.

**Comment by lwaldron** [comment]
BiocSklearn also required a `pip install pandas`, but once Dockerhub is finished building again, BiocSklearn will install on the image.
